variables.tf

```diff
@@ -101,3 +101,23 @@ variable "vm_init_script" {
   type        = "string"
   description = "PowerShell script to initialise the VM"
 }
+variable "domain_name" {
+  type        = "string"
+  description = "Name of the domain to be joinned to"
+}
+variable "OU_path" {
+ type        =  "string"
+ description = "OU to be used for this machine for domain joinning"
+}
+variable "OU_User" {
+  type       = "string"
```

Comment:
the = needs to be indented by one more space on lines 113, 114, 117, 118, 121 and 122